doc/dev/quick_guide.rst

```diff
@@ -25,23 +25,43 @@ Example:
    ./install-deps.sh
    ./do_cmake.sh -DWITH_MANPAGE=OFF -DWITH_BABELTRACE=OFF -DWITH_MGR_DASHBOARD_FRONTEND=OFF
 
+You can also turn off building of some core components that are not relevant to
+your development:
+
+.. prompt:: bash $
+
+   ./do_cmake.sh ... -DWITH_RBD=OFF -DWITH_KRBD=OFF -DWITH_RADOSGW=OFF
+
+Finally, build ceph:
+
+.. prompt:: bash $
+
+   (cd build && ninja -j24)
```

Comment:
@batrick i am all for this change. but have you considered using `cmake --build . [--target <target>...]` instead?

Reply:
I wasn't aware you could do that! I'll make this change.